app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
Personally I'm happy with this as it is. I think @gpeng's argument for readability and clear intent makes sense.